cumulus_library/actions/uploader.py

```diff
@@ -5,22 +5,23 @@
 
 import requests
 from pandas import read_parquet
-from rich.progress import Progress, TaskID
+from rich import console, progress
 
 from cumulus_library import base_utils
 
 
 def upload_data(
-    progress: Progress,
-    file_upload_progress: TaskID,
+    progress_bar: progress.Progress,
+    file_upload_progress: progress.TaskID,
     file_path: Path,
     version: str,
     args: dict,
 ):
     """Fetches presigned url and uploads file to aggregator"""
     study = file_path.parts[-2]
     file_name = file_path.parts[-1]
-    progress.update(file_upload_progress, description=f"Uploading {study}/{file_name}")
+    c = console.Console()
```

Comment:
Is there a meaningful difference between creating your own console object vs the singleton console object from `rich.get_console()`? (here and elsewhere)

Reply:
So - this is mostly a byproduct of rich's documentation - if you're looking at the console API docs, they don't mention this other method existing, so I was unaware of it until now.

The actual functional difference is that the `get_console` method produces a global instance, whereas the rich.console.Console() approach gets a local one. If we cared about style bleed, this might be an issue; I don't believe that we do.

I'm happy to standardize on either style. if it's get_console, i'd take that update as a followon issue (and probably bite the bullet on doing it all in one go).

Reply:
based on our offline chat - #350